PINCE.py

```diff
@@ -1248,6 +1253,11 @@ def comboBox_ScanType_init(self):
         current_type = self.comboBox_ScanType.currentData(Qt.ItemDataRole.UserRole)
         self.comboBox_ScanType.clear()
         items = typedefs.SCAN_TYPE.get_list(self.scan_mode)
+
+        # Delete Not option if Value_Type selected is AOB or STRING
+        if self.comboBox_ValueType.currentData(Qt.ItemDataRole.UserRole) == typedefs.SCAN_INDEX.AOB or self.comboBox_ValueType.currentData(Qt.ItemDataRole.UserRole) == typedefs.SCAN_INDEX.STRING:
+            del items[1]    # Corresponds to NOT in "get_list()" return
```

Comment:
I don't think deletion is the best approach here. Try looking at how `INCREASED`, `DECREASED` and so on are hidden from the user in code and emulate that behaviour.

Reply:
Are you mentioning the `get_list()` function?

From what I can tell `pushButton_NewFirstScan_clicked()` always calls `comboBox_ScanType_init()`. From here `get_list(self.scan_mode)` is called which will return `INCREASED`, `DECREASED` if scan_mode is `ONGOING`.

So, I should modify the `get_list()` function?

Reply:
Yes, you should pass the value type to that function as well so you can check inside and see whether `NOT` should be in or not.

This way it's more portable and we don't do "hacky" deletions in random places in code.

Reply:
Is this better?

```python
    def get_list(scan_mode, value_type):
        list = []
        if scan_mode == SCAN_MODE.NEW:
            list = [
                SCAN_TYPE.EXACT,
                SCAN_TYPE.NOT,
                SCAN_TYPE.LESS,
                SCAN_TYPE.MORE,
                SCAN_TYPE.BETWEEN,
                SCAN_TYPE.UNKNOWN,
            ]
        else:
            list = [
                SCAN_TYPE.EXACT,
                SCAN_TYPE.NOT,
                SCAN_TYPE.INCREASED,
                SCAN_TYPE.INCREASED_BY,
                SCAN_TYPE.DECREASED,
                SCAN_TYPE.DECREASED_BY,
                SCAN_TYPE.LESS,
                SCAN_TYPE.MORE,
                SCAN_TYPE.BETWEEN,
                SCAN_TYPE.CHANGED,
                SCAN_TYPE.UNCHANGED,
            ]

        # Delete Not option
        if value_type == SCAN_INDEX.AOB or value_type == SCAN_INDEX.STRING:
            del list[1]
        
        return list
```

Reply:
Yes, get rid of the comment, it's useless.

Reply:
Ok, thanks!